model/commitq/githubpr/github_pr_sender.go

```diff
@@ -0,0 +1,97 @@
+package githubpr
+
+import (
+	"context"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/evergreen-ci/evergreen/model/commitq"
+	"github.com/google/go-github/github"
+	"github.com/mongodb/grip/level"
+	"github.com/mongodb/grip/message"
+	"github.com/mongodb/grip/send"
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+)
+
+type githubPRLogger struct {
+	prService    *github.PullRequestsService
+	statusSender send.Sender
+	*send.Base
+}
+
+// NewGithubPRLogger creates a new Sender implementation that
+// merges a pull request
+// Specify an OAuth token for GitHub authentication
+func NewGithubPRLogger(name string, token string, statusSender send.Sender) (send.Sender, error) {
+	ctx := context.TODO()
+	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: token})
+	tc := oauth2.NewClient(ctx, ts)
+	githubClient := github.NewClient(tc)
```

Comment:
does this create an http.Client? we should make sure we're pooling appropriately.

Reply:
Done.